CHANGELOG.md

```diff
@@ -3,6 +3,13 @@ All notable changes to libcosim will be documented in this file. This includes n
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+### [v0.10.5] - 2025-03-31
+##### New
+* The ECCO (Energy Conservation-based Co-simulation) algorithm has received a reference implementation that is now configurable through OspSystemStructure. See the tests prefixed with `ecco` for details. 
+* Libcosim now supports the state saving API from FMI2. FMUs that support this feature should flag it by setting the `canGetAndSetFMUstate` and `canSerializeFMUstate` fields in their model description to `true`. See the tests prefixed with `state_saving` for details.
```

Comment:
Maybe worth mentioning that we also support serialisation to the CBOR format.

Reply:
Good idea, updated the text accordingly. 

Also changed to v0.11.0 as requested.